access/jira/client.go

```diff
@@ -39,6 +39,9 @@ const (
 
 	jiraMaxConns    = 100
 	jiraHTTPTimeout = 10 * time.Second
+	// Jira has a 300,000 character limit for the reason field so we
+	// truncate all reasons to a generous but conservative limit
+	jiraReasonLimit = 3000
```

Comment:
If Jira has a limit of 300K, why do we limit to 3K?

Reply:
30k sounds good to me.

Reply:
The actual limit appears to be much smaller than what was listed in the ticket. Seemingly 4096 instead of 300000, I have updated the comment to reflect this.